dbschema/project.esdl

```diff
@@ -214,6 +214,26 @@ module Project {
     DidNotDevelop,
   >;
   
+  scalar type TransitionType extending enum<
+    Neutral,
+    Approve,
+    `Reject`,
+  >;
+
+  # Not sure if this is correct since TransitionName is not an enum, see transitions.ts file
+  scalar type TransitionName extending enum<
+    `Start`,
+    `In Progress -> Pending Translation`,
+    `In Progress -> In Review`,
+    `Translation Done`,
+    `Translation Reject`,
+    `Withdraw Review Request`,
+    `In Review -> Needs Translation`,
+    `Review Reject`,
+    `Review Approve`,
+    Publish,
+  >;
+
```

Comment:
I think these make the most sense inside the Workflow Transition type.  They would then be part of module, so could be renamed and reference like `WorkflowTransition.name` & `WorkflowTransition.type`.